hnn_core/network_builder.py

```diff
@@ -415,17 +415,19 @@ def _create_cells_and_feeds(self, threshold, record_vsoma=False,
                 _PC.cell(drive_cell.gid, drive_cell.nrn_netcon)
                 self._drive_cells.append(drive_cell)
 
-    def _connect_celltypes(self, src_type, target_type, loc,
+    def _connect_celltypes(self, src, target, loc,
                            receptor, nc_dict, unique=False,
                            allow_autapses=True):
         """Connect two cell types for a particular receptor.
 
         Parameters
         ----------
-        src_type : str
-            The source cell type
-        target_type : str
-            The target cell type
+        src : tuple of (list of str, list of int)
```

Comment:
any reason for it to be immutable tuple? if not, I'd suggest keeping it simple and make it list of list

Reply:
Good point, went with lists for the new `connectivity` variable that is passed into `connect_celltypes()`